topics/introduction/tutorials/galaxy-intro-101/tutorial.md

```diff
@@ -58,23 +58,36 @@ We start with the question: In human chromosome 22, which exon has the highest n
+> **SNP** is an abbreviation for [single-nucleotide polymorphism](https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism). It is pronounced as "snip". SNPs are single nucleotide differences between a sequenced individual compared to some reference sequence; where one individual might have an A, another could have a C in that position. Databases of SNPs have been created for many organisms and they include any single nucleotide deviation from the reference sequence which appears in some percentage of the population (e.g. >1%). These are often especially interesting to geneticists as the causes of certain inheritable diseases.
 {: .comment}
 
 You may be familiar with the [UCSC Genome Browser](https://genome.ucsc.edu/) or another resource like it, and know that you can find the data there.
-But even with your data in hand, you stll have the question: "how do I actually compute this?" There is really a straightforward answer: **Galaxy**. So let's try it...
+But even with your data in hand, you still have the question: "how do I actually compute this?" There is really a straightforward answer: **Galaxy**. So let's try it...
+
+The following video shows all steps of this tutorial:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/D5HgJWdfOWw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+Browse to your favorite [Galaxy instance](https://galaxyproject.org/use/) and log in or register. 
 
-Browse to your favourite [Galaxy instance](https://galaxyproject.org/use/) and log in or register. The Galaxy interface consists of three main parts:
+{% snippet faqs/galaxy/creating_account.md %}
 
-1. The available tools are listed on the left
-2. Your analysis history is recorded on the right
-3. The central panel will let you run analyses and view outputs
+The Galaxy interface consists of three main parts:
 
-![Galaxy interface](../../images/galaxy_interface.png)
+1. The available **TOOLS** are listed on the left
+2. The **CENTRAL PANEL** will let you run analyses and view outputs
+3. Your analysis **HISTORY** is recorded on the right
+
+
+> ![Galaxy interface](../../images/galaxy_interface.svg "Galactic triptych: the three panels of Galaxy interface: TOOLS, CENTER PANEL, and HISTORY")
```

Comment:
```suggestion
> ![Galaxy interface](../../images/galaxy_interface.svg "Galactic triptych: the three panels of Galaxy interface: <em>Tools</em>, <em>Center Panel</em>, and <em>History</em>")
```

I don't think the SVG has sufficient control for the Center Panel text, could you change that to red or a darker colour please?